clients/mssql/staging.go

```diff
@@ -0,0 +1,69 @@
+package mssql
+
+import (
+	"fmt"
+
+	mssql "github.com/microsoft/go-mssqldb"
+
+	"github.com/artie-labs/transfer/lib/config/constants"
+	"github.com/artie-labs/transfer/lib/destination/ddl"
+	"github.com/artie-labs/transfer/lib/destination/types"
+	"github.com/artie-labs/transfer/lib/optimization"
+)
+
+func (s *Store) PrepareTemporaryTable(tableData *optimization.TableData, tableConfig *types.DwhTableConfig, tempTableName string, additionalSettings types.AdditionalSettings) error {
+	tempAlterTableArgs := ddl.AlterTableArgs{
+		Dwh:               s,
+		Tc:                tableConfig,
+		FqTableName:       tempTableName,
+		CreateTable:       true,
+		TemporaryTable:    true,
+		ColumnOp:          constants.Add,
+		UppercaseEscNames: &s.config.SharedDestinationConfig.UppercaseEscapedNames,
```

Comment:
Is it possible to not use `UppercaseEscNames` for MSSQL?

Reply:
Definitely possible, but I don't think the trade off of reducing migration scope is worth the code fragmentation